pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -60,7 +60,7 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 
 	providers, err := s.loadbalancerClient.ListLoadBalancerProviders()
 	if err != nil {
-		return err
+		return true, err
```

Comment:
I'm not sure if this should be a terminal error. What if there was some temporary network issue?

Reply:
right ,I think should make it false